qemu/qemu.go

```diff
@@ -128,6 +128,10 @@ const (
 func isDimmSupported(config *Config) bool {
 	switch runtime.GOARCH {
 	case "amd64", "386":
+		if config != nil && config.Machine.Type == "microvm" {
```

Comment:
can you define a constant for `microvm`?

Reply:
Sure.  I added MachineTypeMicrovm